docs-generation/scripts/update_param_translations.py

```
@@ -18,16 +24,76 @@
+    for lang in FETCH_LANGS:
+        url = f"{REMOTE_BASE}/{lang}/nodeDefs.json"
+        try:
+            req = urllib.request.Request(url, headers={"User-Agent": "docs-generation"})
+            with urllib.request.urlopen(req, timeout=30) as resp:
+                out[lang] = json.loads(resp.read().decode("utf-8"))
+        except Exception as e:
+            print(f"⚠️  Warning: failed to fetch {lang} from GitHub ({url}): {e}")
+            out[lang] = {}
+    return out
+
+def load_frontend_translations(force_refresh=False):
```

Ruff (0.16.1)

[warning] Boolean default positional argument in function definition

(FBT002)